v2/api/compute/v1beta20220301/structure.txt

```diff
@@ -1570,7 +1573,10 @@ github.com/Azure/azure-service-operator/v2/api/compute/v1beta20220301
 │   │   │   │   │   ├── ForceUpdateTag: *string
 │   │   │   │   │   ├── Name: *string
 │   │   │   │   │   ├── ProtectedSettings: map[string]v1.JSON
-│   │   │   │   │   ├── ProtectedSettingsFromKeyVault: map[string]v1.JSON
+│   │   │   │   │   ├── ProtectedSettingsFromKeyVault: *Object (2 properties)
```

Comment:
This structural change was made by compute here: https://github.com/Azure/azure-rest-api-specs/pull/20101

I think it's safe for us as well since the acceptable JSON structure is the same - they just clarified they only allow certain properties.